### ISSUE TITLE
User credidentials should be encrypted

### ISSUE THREAD

**Issue by knikolov-nuvolo** [nuvolo/sincronia#69]
The `.env` file contains Instance URL, Username and Password, all in plain text.
The file is in `.gitignore` so it is not committed, but this do not prevent unauthorized people to get its content (e.g. accidentally sending the file together with other files; during a screen share; hacker attack; malicious software; stolen device, etc.)

The information in the file can give access to the given instance together with all the source code, configurations, data, etc.
Personally I think this is a huge security hole.

Possible solutions are:
1. Enforce encryption on all configuration files. The encryption key can be some of the following (or combination):
  a. Custom, by the user - must be entered every time he/she runs a command;
  b. Custom, by the user - saved in separate configuration file outside of the project (e.g. `~/.config/sincronia`);
  c. Auto-Generated based on Computer name, IP address (must be static), MAC address, or something else device/user specific;
  d. Hard coded in the source code (not recommended, since this is an open source project).
2. Don't keep the password in the file, and ask for it every time.

For best experience (avoid entering a password every time) I think that some combination of 1.b. and 1.c. would be the best. But any suggestions are welcome.

**Comment by nabil-nuvolo** [comment]
@knikolov-nuvolo  what do you think about using oauth instead? 

Granted we should probably still encrypt the access/refresh tokens but possibly it is less severe if leaked

**Comment by knikolov-nuvolo** [comment]
That would be a better solution. Tokens can be easily revoked if leaked.

But I still think we should be able to keep authentication details in central place outside of the project. This will prevent accidentally shearing the token with someone.
Also you won't be required to authenticate for the same instance on every application.
But this might be a separate task.

**Comment by nabil-nuvolo** [comment]
Agree, perhaps a recursive search through directories until a .env file is found with the proper params?
